app/(gcforms)/[locale]/(form administration)/form-builder/[id]/settings/manage/actions.ts

```diff
@@ -69,16 +69,40 @@ export const resetThrottlingRate = AuthenticatedAction(async (session, formId: s
   }
 });
 
-export const updateNotificationsInterval = AuthenticatedAction(
-  async (_, formId: string, notificationsInterval: NotificationsInterval) => {
+export const saveNotificationsSettings = AuthenticatedAction(
+  async (_, formId: string, user: { id: string; email: string; enabled: boolean } | null) => {
```

Comment:
> // A user can only update their own notifications settings

Given a user can only update there own settings  - what is the need to pass the user here ? We confirm the user when the settings is updated so might as well not pass and just pull from the session itself no?